app/views/courses/preview/_financial_incentives.html.erb

```diff
@@ -0,0 +1,9 @@
+<% if course.salaried? %>
+  <%= render partial: 'courses/preview/financial_support/salaried' %>
+<% elsif course.bursary_only? %>
+  <%= render partial: 'courses/preview/financial_support/bursary' %>
+<% elsif course.has_scholarship_and_bursary? %>
+  <%= render partial: 'courses/preview/financial_support/scholarship_and_bursary' %>
+<% else %>
+  <%= render partial: 'courses/preview/financial_support/loan' %>
```

Comment:
Boilerplate for all but salaried

```ruby 
if salaried?
  ...
else
  if
    ...
  else
    ...
  end
#boilerplating
	
end
```

Reply:
Salaried partial is on line 2

```
<% if course.salaried? %>
  <%= render partial: 'courses/preview/financial_support/salaried' %>
<% elsif course.bursary_only? %>
  <%= render partial: 'courses/preview/financial_support/bursary' %>
<% elsif course.has_scholarship_and_bursary? %>
  <%= render partial: 'courses/preview/financial_support/scholarship_and_bursary' %>
<% else %>
  <%= render partial: 'courses/preview/financial_support/loan' %>
<% end %>
```

I'm assuming the above is ok as a non-salaried course would not have a bursary or scholarship right?